package.json

```
@@ -7,6 +7,7 @@
   "scripts": {
     "start": "node app",
     "lint": "eslint . --max-warnings 0",
+    "lint-fix": "eslint . --fix",
```

Could we match the syntax in actual here?

```suggestion
    "lint:fix": "eslint . --fix",
```